src/media/js/mobilenetwork.js

```diff
@@ -346,10 +38,6 @@ define('mobilenetwork',
             }
         }
 
-        // Make them integers.
-        mcc = +mcc || 0;
```

Comment:
No, as long as it matches the keys in `carrierRegions` we don't care. When moving them to `marketplace-constants` I just left the default behaviour, which is to format the objects as json, and they ended up being strings, so I removed the conversion to integers here.